package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@neo4j/docs-ui",
-  "version": "0.29.7",
+  "version": "0.29.8",
```

Comment:
You don't need to update this here. It will be updated automatically when you run `npm version patch` as part of the release process.